src/solver/algorithm/strong/cyclic.rs

```diff
@@ -6,3 +6,379 @@
 //!
 //! #### Authorship
 //! - Max Fierro, 12/3/2023 (maxfierro@berkeley.edu)
+//! - Ishir Garg, 3/12/2024 (ishirgarg@berkeley.edu)
+
+use anyhow::{Context, Result};
+use std::collections::{HashMap, VecDeque};
```

Comment:
New line missing between external imports and `std` imports. Thanks for separating these though.